crates/rooch/src/commands/account/commands/binding.rs

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::cli_types::{TransactionOptions, WalletContextOptions};
+use bitcoincore_rpc::{Auth, Client, RpcApi};
+use clap::Parser;
+use rooch_types::{
+    address::{BitcoinAddress, MultiChainAddress, RoochAddress},
```

Comment:
Isn't it included BitcoinAddress to the MultiChainAddress? We were using only two types of addresses before.